skywalking-sniffer/skywalking-api/src/main/java/com/a/eye/skywalking/util/ContextGenerator.java

```diff
@@ -42,12 +42,12 @@ private static Span getSpanFromThreadLocal(Identification id) {
         // 2 校验Context，Context是否存在
         if (parentSpan == null) {
             // 不存在，新创建一个Context
-            span = new Span(id.getViewPoint());
+            span = new Span(id.getViewPoint(), id.getStartTime(), id.getTags());
```

Comment:
`Tags`, as a  constructor parameters, is an improper thing.
`Span` should have a method, named `tag()`